### PR TITLE
Update payment sessions schemas

### PR DESCRIPTION
**Pull request by armando-rodriguez-cko**
- Updates payment sessions schemas

### CODE REVIEW

File: src/CheckoutSdk/Common/CardholderAccountAgeIndicatorType.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+namespace Checkout.Common
+{
+    public enum CardholderAccountAgeIndicatorType
+    {
+        [EnumMember(Value = "less_than_thirty_days")]
+        LessThanThirtyDays,
+
+        [EnumMember(Value = "more_than_sixty_days")]
+        MoreThanSixtyDays,
+        [EnumMember(Value = "no_account")] NoAccount,
```
Comment (robert-goheen-cko) [inline src/CheckoutSdk/Common/CardholderAccountAgeIndicatorType.cs:12]: Is this a correct enumerator?

File: src/CheckoutSdk/Sessions/ChallengeCancelReason.cs [hunk 2]
```diff
@@ -7,15 +7,17 @@ public enum ChallengeCancelReason
         [EnumMember(Value = "cardholder_cancel")]
         CardholderCancel,
 
-        [EnumMember(Value = "transaction_timed_out")]
-        TransactionTimedOut,
-
         [EnumMember(Value = "challenge_timed_out")]
         ChallengeTimedOut,
+        
+        [EnumMember(Value = "sdk_timed_out")] SdkTimedOut,
```
Comment (robert-goheen-cko) [inline src/CheckoutSdk/Sessions/ChallengeCancelReason.cs:13]: Can you put this on two lines like the other entries?

Comment (robert-goheen-cko) [reply src/CheckoutSdk/Sessions/ChallengeCancelReason.cs:13]: I'm really not sure what the difference is between SDK timed out and transaction timed out

File: src/CheckoutSdk/Common/CardholderAccountAgeIndicatorType.cs (hunk 1 above)
Comment (armando-rodriguez-cko) [reply src/CheckoutSdk/Common/CardholderAccountAgeIndicatorType.cs:12]: I have attached here: https://api-reference.checkout.com/#operation/createSession!path=account_info/account_password_change_indicator&t=request. I will format it correctly

File: src/CheckoutSdk/Sessions/ChallengeCancelReason.cs (hunk 2 above)
Comment (armando-rodriguez-cko) [reply src/CheckoutSdk/Sessions/ChallengeCancelReason.cs]: 👍🏼

**Review by robert-goheen-cko — APPROVED** [review]

**Pull request merged**